docs/usage.md

```diff
@@ -134,20 +139,49 @@ A Zarr array is defined not just by the location of its constituent chunk data,
 ```python
 marr
 ```
+
 ```
 ManifestArray<shape=(2920, 25, 53), dtype=int16, chunks=(2920, 25, 53)>
 ```
+
 ```python
 marr.manifest
 ```
+
 ```
 ChunkManifest<shape=(1, 1, 1)>
 ```
+
 ```python
-marr.zarray
+marr.metadata
 ```
+
 ```
-ZArray(shape=(2920, 25, 53), chunks=(2920, 25, 53), dtype=int16, compressor=None, filters=None, fill_value=None)
+ArrayV3Metadata(shape=(2920, 25, 53),
+                data_type=<DataType.float64: 'float64'>,
+                chunk_grid=RegularChunkGrid(chunk_shape=(2920, 25, 53)),
+                chunk_key_encoding=DefaultChunkKeyEncoding(name='default',
+                                                           separator='/'),
+                fill_value=np.float64(-327.67),
```

Comment:
Seeing the full representation I looked into why the fill value was `np.float64(-327.67)` as that fill value was not in the xarray encoding but was the fill value of the `h5py.Dataset`, is it worth digging into why that float value wasn't in the xarray encoding? @sharkinsspatial

Reply:
Discussing this in a separate thread, will follow up once resolved...

Reply:
The outcome of this side conversation was that we have not yet resolved https://github.com/zarr-developers/VirtualiZarr/issues/414 via https://github.com/zarr-developers/VirtualiZarr/pull/420 and that the way to do this is by including the HDF5 attribute `_FillValue` in the Xarray Variable's encoding. And then when writing the dataset to icechunk, it will be included as an attr (see https://github.com/zarr-developers/VirtualiZarr/blob/c245b0a5c2d1c6c4eba22557b22c497473284853/virtualizarr/writers/icechunk.py#L263-L266)